tests/conftest.py

```diff
@@ -48,12 +49,57 @@ def fixture_code(fixture_localhost):
     """Return a `Code` instance configured to run calculations of given entry point on localhost `Computer`."""
 
     def _fixture_code(entry_point_name):
+        from aiida.common import exceptions
         from aiida.orm import Code
-        return Code(input_plugin_name=entry_point_name, remote_computer_exec=[fixture_localhost, '/bin/true'])
+
+        label = f'test.{entry_point_name}'
+
+        try:
+            return Code.objects.get(label=label)  # pylint: disable=no-member
+        except exceptions.NotExistent:
+            return Code(
+                label=label,
+                input_plugin_name=entry_point_name,
+                remote_computer_exec=[fixture_localhost, '/bin/true'],
+            )
 
     return _fixture_code
 
 
+@pytest.fixture(scope='session', autouse=True)
+def sssp(aiida_profile, generate_upf_data):
+    """Create an SSSP pseudo potential family from scratch."""
+    from aiida.common.constants import elements
+    from aiida.plugins import GroupFactory
+
+    aiida_profile.reset_db()
+
+    SsspFamily = GroupFactory('pseudo.family.sssp')
+
+    cutoffs = {'standard': {}}
+
+    with tempfile.TemporaryDirectory() as dirpath:
+        for values in elements.values():
+
+            element = values['symbol']
+            upf = generate_upf_data(element)
+            filename = os.path.join(dirpath, f'{element}.upf')
+
+            with open(filename, 'w+b') as handle:
+                with upf.open(mode='rb') as source:
+                    handle.write(source.read())
+                    handle.flush()
```

Comment:
Just out of curiosity: why do we `flush` the file handle here?

Reply:
just to make sure it is written to disk, but I think it is not actually necessary here, since by the time the file will be used, it is outside of this context manager, the closing of which should guarantee the flushing as well.